source/javascripts/components/_AddStep.js.erb

```diff
@@ -127,6 +127,7 @@ angular.module("BitriseWorkflowEditor").service("addStepService", function() {
 						return aLibrary.steps[stepID][aStepVersion].isDeprecated();
 					}), function(aStepVersion, stepID) {
 						var step = angular.copy(aLibrary.steps[stepID][aStepVersion]);
+						step.info = angular.copy(aLibrary.steps[stepID].info)
```

Comment:
If I understand it correctly, this is not a changeable property of a step. So it is okay that by reference it is the same object for two step instances since we won't be changing any of them. The copy one line above is necessary because it contains changeable properties.
```suggestion
						step.info = aLibrary.steps[stepID].info;
```